modules/nf-core/oncocnv/main.nf

```diff
@@ -0,0 +1,75 @@
+
+process ONCOCNV {
+    tag "$tumor_dataset_id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::oncocnv=7.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/oncocnv/v7.0_cv1/oncocnv_v7.0_cv1.sif':
+        'registry.hub.docker.com/biocontainers/oncocnv:v7.0_cv1' }"
+
+    input:
+    tuple val(normal_dataset_id), path(normal_bams), path(normal_bais)
+    tuple val(tumor_dataset_id), path(tumor_bams), path(tumor_bais)
+    path bed
+    path fasta
+
+    output:
+    tuple path("*.profile.png")  ,emit: png
+    tuple path("*.profile.txt")  ,emit: profile
+    tuple path("*.summary.txt")  ,emit: summary
+    path "versions.yml"          ,emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def cghseg = task.ext.cghseg ? 'cghseg' : ''
+    def mode = task.mode ?: 'Ampli'
```

Comment:
These look like text based arguments, if that is the case they will both go into `task.ext.args`

Reply:
It might be a problem for me as I am calling different commands which require different arguments. `task.ext.args` looks like everything is lumped together but I might be wrong...

Reply:
I guess I could define two of these?